K2Bridge.Tests.End2End/TestElasticClient.cs

```diff
@@ -253,8 +232,11 @@ private static void NormalizeIndexNamesForIndexList(JToken parent, string jsonPa
             {
                 if (tokens[index].Value.Equals(indexName))
                 {
-                    var tableName = indexName.Split(':')[1];
-                    tokens[index].Value = tableName;
+                    if (indexName.Contains(':'))
```

Comment:
I think that  you are correct and the ':' must exists. did you made sure that its the case?

Reply:
I am now calling this function also on the Elastic results (to prune unwanted indices), in which case the index name has no ':' sign.

https://github.com/microsoft/K2Bridge/blob/feature/kibana-7-resolve-index/K2Bridge.Tests.End2End/ParallelApiTest.cs#L169

Reply:
Reverted (see above).